Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml

```diff
@@ -1670,10 +1670,13 @@
   - type: SolutionContainerManager
     solutions:
       drink:
-        maxVol: 30
+        maxVol: 20
         reagents:
         - ReagentId: Sake
-          Quantity: 30
+          Quantity: 20
```

Comment:
why did sake get reduced to 20u?

Reply:
Because it's a tiny drink?

Reply:
Can you put it back to 30 again? Standardisation is better here.